neuromation/api/parsing_utils.py

```diff
@@ -158,14 +165,29 @@ def _split_image_name(
     ) -> Tuple[str, Optional[str]]:
         colon_count = image.count(":")
         if colon_count == 0:
-            image, tag = image, default_tag
+            name, tag = image, default_tag
         elif colon_count == 1:
-            image, tag = image.split(":")
-            if not tag:
+            # case `ubuntu:latest`
+            name, tag = image.split(":")
+            if "/" in tag:
+                # case `localhost:5000/ubuntu`
+                name, tag = image, default_tag
+            elif not tag:
                 raise ValueError("empty tag is not allowed")
+        elif colon_count == 2:
+            # case `localhost:9000/owner/ubuntu:latest`
+            if "/" not in image:
+                # case `ubuntu:v11:latest`
+                raise ValueError("too many tags")
+            *image_arr, tag = image.split(":")
+            name = ":".join(image_arr)
         else:
             raise ValueError("too many tags")
-        return image, tag
+        if not tag:
+            tag = default_tag
```

Comment:
"localhost/owner/ubuntu:" raises an error, but "localhost:9000/owner/ubuntu:" uses the default tag.

Reply:
considered, thanks